system/core/HyphaRequest.php

```diff
@@ -149,11 +145,7 @@ public function getScheme() {
 
 		public function getHost() {
 			return $_SERVER['HTTP_HOST'];
-		}
-
-		public function getPort() {
```

Comment:
I would not remove getPort entirely, other parts of the code might still want to query the port number.